src/components/sections/radio-section.tsx

```diff
@@ -2,23 +2,28 @@ import React from 'react';
 import ActionItem from './action-item';
 import SectionGroup, {SectionProps} from './section';
 import {HeaderItem} from '@atb/components/sections/index';
+import {InteractiveColor} from '@atb/theme/colors';
 
 export type RadioSectionProps<T> = Omit<SectionProps, 'type' | 'children'> & {
   items: T[];
   selected?: T;
   keyExtractor(item: T, index: number): string;
   itemToText(item: T, index: number): string;
+  itemToSubtext?(item: T, index: number): string;
   onSelect?(item: T, index: number): void;
   headerText?: string;
+  color?: InteractiveColor;
 };
 
 export default function RadioSectionGroup<T>({
   keyExtractor,
   itemToText,
+  itemToSubtext,
   items,
   selected,
   onSelect,
   headerText,
+  color,
   ...props
```

Comment:
> Paddings on single traveler selection are inconsistent with sketches. Need to figure out if it's intended that the sketch paddings are applied to radio boxes other places in the app.

`ActionItem` and all other sections items have a parameter `type`, which can have value 'spacious' for larger padding. This parameter can be exposed on the `RadioSectionGroup`.

Reply:
Not sure why this was not done here previously, but an easy fix. I just had to **not** omit it from the props. 4e2bc7a73c98ea17612dbca02139b1de46245a69